exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/MetricAdapter.java

```diff
@@ -58,12 +58,15 @@
           new ArrayList<>(entryResource.getValue().size());
       for (Map.Entry<InstrumentationLibraryInfo, List<Metric>> entryLibrary :
           entryResource.getValue().entrySet()) {
-        instrumentationLibraryMetrics.add(
+        InstrumentationLibraryMetrics.Builder metricsBuilder =
```

Comment:
Small suggestion - with extra "if" in the builder flow, perhaps it makes sense to move the Builder creation/configuration to a separate (helper) method? Then parent "for" flow will be a slightly more readable (for (....) instrumentationLibraryMetrics.add(metricsBuilderFor(entryLibrary).build())